ibis/backends/pyspark/__init__.py

```diff
@@ -929,3 +940,110 @@ def to_pyarrow_batches(
         return pa.ipc.RecordBatchReader.from_batches(
             pa_table.schema, pa_table.to_batches(max_chunksize=chunk_size)
         )
+
+    @util.experimental
+    def read_kafka(
+        self,
+        table_name: str | None = None,
+        watermark: Watermark | None = None,
+        auto_parse: bool = False,
+        schema: sch.Schema | None = None,
+        options: Mapping[str, str] | None = None,
+    ) -> ir.Table:
+        """Register a Kafka topic as a table in the current database.
```

Comment:
```suggestion
        """Register a Kafka topic as a table.
```

The fact that the table is in the current database doesn't seem particularly informative here.